src/types/aws/ap/registration.ts

```diff
@@ -146,7 +152,9 @@ export type SavedRegistration =
   | DoneContact
   | DoneOrgDetails
   | DoneFSAInquiry
+  | ResetFSAInquiry
```

Comment:
@SovereignAndrey, reverted most of the changes - figured that instead of modifying existing `Done` states, `SaveRegistration` has now reset variants